Leaflet.Graticule.js

```diff
@@ -193,13 +193,30 @@ L.LatLngGraticule = L.Layer.extend({
         }
 
         // todo: format type of float
+        var dir = '';
+
         if (lat < 0) {
-            return '' + (lat*-1) + 'S';
+            lat = lat*-1;
+            dir = 'S';
         }
         else if (lat > 0) {
-            return '' + lat + 'N';
+            dir = 'N';
         }
-        return '' + lat;
+
+        var zoom = this._map.getZoom();
+        lat = '' + lat;
+
+        if(lng.indexOf(".") != -1) {
+          if(zoom >= 9 && zoom < 13)
```

Comment:
So here I still think you should not use zoom as a way to determine the precision -- because people can configure that differently. 

Instead use:

```js
var decimalPlacesAfterZero = 0; if (lat < 1) placesAfter = (lat + '').split('.')[1].length;
```

To measure what precision was used in the configuration, and trim everything accordingly. Make sense?